cli/src/utils/getdeps.ts

```diff
@@ -46,7 +50,7 @@ export async function getDependencies(projectDir: string) {
     if (dependencies.has(p)) continue;
 
     if (lockHasProperty(p)) {
-      dependencies.add(p);
+      dependencies.set(p, { optional: Boolean(lockDeps[p].optional) });
```

Comment:
Boolean can be written as `!!`?

Reply:
I prefer `Boolean`.

Reply:
You could always use the `[true][+...]` operator:
```js

> [true,false][+'']
true
> [true,false][+2]
undefined
> [true,false][+'s']
undefined
> [true][+'s']
undefined
> [true][+'']
true
> [true][+false]
true
> [true][+true]
undefined
> [true][+0]
true
> [true][+-2]
undefined
>
```